### PR TITLE
Ft agefieldon registration

### PR DESCRIPTION
**Pull request by BrendaNamutebi**
PR Checklist (Make sure these options are checked before raising the PR)

- [ ] - Latest `master` branch merged
- [ ] - Descriptive PR title (can be used in release notes)

#### Summary of Changes (What does this PR do?)
- Changed the format of capturing the user's age, by capturing the user's date of birth

#### Is this change ready to hit production in its current state?
- [x] Yes
- [ ] No

#### Jira Ticket(s) Number
- [SA-70]

## Screens
DOB
hots/Videos

_If useful, provide screenshot or capture to highlight main changes_

### CODE REVIEW

File: src/config/urls.js [hunk 1]
```diff
@@ -4,3 +4,5 @@ export const SEIZURE_URL = `${BASE_URL}/seizures`;
 export const RESILIENCE_URL = `${BASE_URL}/resilience`;
 export const REGISTER_USER_URL = `${BASE_URL}/users`;
 export const LOGIN_USER_URL = `${BASE_URL}/users/login`;
+// export const USER_ACTIVITIES = `${BASE_URL}/users`; // appended with an id
```
Comment (KingMike100) [inline src/config/urls.js:7]: You can clean this and get rid of this added comment

File: src/pages/Authentication/Register.jsx [hunk 2]
```diff
@@ -90,14 +90,12 @@ const Register = () => {
         </div>
         <div className="form-select-group">
           <div>
-            <label htmlFor="age">Age</label>
+            <label htmlFor="age">Date of birth</label>
```
Comment (KingMike100) [inline src/pages/Authentication/Register.jsx:93]: Have you tested this on the backend by creating a new user with a date of birth? I am failing whenever I add one. API returns "age could not be null". Either @MutegekiHenry or Derrick have not yet implemented that route. Confirm if this is the case

**Comment by KingMike100** [comment]
Nice attempt sofar, I'm unable to register a new user with the date of birth field. Have you tested this on your end? I will also cross-check with Henry and Derrick

File: src/pages/Authentication/Register.jsx (hunk 2 above)
Comment (BrendaNamutebi) [reply src/pages/Authentication/Register.jsx:93]: Backend people hadn't yet implemented the  route

**Pull request closed without merging**